pkg/util/policy/agent/agent.py

```diff
@@ -185,9 +265,9 @@ def policy_update(romana_address_scheme, policy_definition, delete_policy=False)
     implemented here. TODO!
 
     """
-
+    
     # PolicyId is a uniq tag that we are going to use to check if rule is applied already
-    policy_id      = "%s/%s" % ( policy_definition['policy_name'], policy_definition['policy_namespace'] )
+    policy_id      = policy_definition['name']
```

Comment:
I was thinking about that. Right now we allow policies without UUID, do we want to disallow it ?